### DISCUSSION TITLE
how to update zoom

### DISCUSSION THREAD

**Discussion by subhasispanda98** [stephanrauh/ngx-extended-pdf-viewer#1027] [Q&A]
my default zoom is 'page-fit'. But when the user clicks on the zoom in, out button get the current zoom value from page-fit and updates zoom to that value. Read your docs can't get it.

**Comment by stephanrauh** [comment]
Just double-checking I get you right: Let's assume `page-fit` amounts to 111%. Clicking one of the zoom buttons replaces the `page-fit` setting by `111%`? 

My assumption is it uses either 100% (zoom out) or 120% (zoom in). Everything else is a surprise to me.

**Reply by subhasispanda98** [reply]
if page-fit 111% then if I click to zoom in button it will update the zoom value from 111% to a certain value. how to do that, how to get current zoom value

**Reply by stephanrauh** [reply]
Are you describing the bug that also visible at https://pdfviewer.net/extended-pdf-viewer/two-way-binding ? By default, the showcase page uses the "auto" zoom setting, and the two-way-bound input field also shows "auto". After clicking the "-" once, the input field still shows "auto", although the setting in the toolbar is already "100%". When you click "-" again, the two-way bound input field shows "90%". Only the first click after using one of the non-numeric zoom setting does not synchronize the zoom setting.

That's a bug I didn't create an issue for yet.

**Reply by stephanrauh** [reply]
A workaround is to use `(currentZoomFactor)`. AFAIR this event fires reliably.

**Reply by subhasispanda98** [reply]
yes thank you (currentZoomFactor) solved the issue

**Comment by ibtissemoussaifi** [comment]
Hello,
With book mode, when i try to zoomIn the page, it hides under the other and i can't see the whole page
Can you help me , how can i fix it ?
thanks.
Best Regards.

**Reply by stephanrauh** [reply]
I'm afraid it's my job to fix it. It sounds like a bug of the library. Of course, the idea of open source software is everyone can contribute, so I'd appreciate your help. However, in this particular case, it's probably difficult.

How can I reproduce the problem? Does it happen on the showcase? If so, can you open a bug ticket, please?

**Reply by ibtissemoussaifi** [reply]
Yes, it happens in the showcase, i opened a bug ticket
thanks.

**Reply by stephanrauh** [reply]
Awesome, thanks!
